fontforge/python.c

```diff
@@ -2807,42 +2929,35 @@ Py_RETURN( self );
 static PyObject *PyFFContour_InsertPoint(PyFF_Contour *self, PyObject *args) {
     double x,y;
     PyFF_Point *p=NULL;
-    int i, on, pos;
+    int i, on, pos, sel;
 
-    x = y = 0.0; pos = -1; on = true;
-    if ( !PyArg_ParseTuple( args, "(ddi)|i", &x, &y, &on, &pos )) {
+    x = y = 0.0;
+    pos = -1;
+    on = true;
+    sel = false;
+    if ( !PyArg_ParseTuple( args, "(ddii)|i", &x, &y, &on, &sel, &pos )) {
 	PyErr_Clear();
-	if ( !PyArg_ParseTuple( args, "(dd)|ii", &x, &y, &on, &pos )) {
```

Comment:
This is the most obvious aspect of the old interface that I broke, and I broke it because it was crazy. Did the user of the previous interface mean to specify on-curve or pos? Who knows? Just leave the point spec in a tuple and the pos outside.